incubator/zookeeper/templates/service-headless.yaml

```diff
@@ -13,6 +13,7 @@ metadata:
 {{- end }}
 spec:
   clusterIP: None
+  publishNotReadyAddresses: true
```

Comment:
For community charts this should be a values config instead of always coded into the template, since it seems it isn’t an issue for all users or all or of the time.

Alternatively Helm 3 now has the ability to add options like this with a post-render hook (see “post rendering” at https://helm.sh/docs/topics/advanced/). However we’re still within the helm 2 support window, so if this were configurable this PR would be a good addition.

Reply:
Added option to values.yaml